backend/app/__init__.py

```diff
@@ -83,3 +83,88 @@ def one_item(item_id):
 @app.route("/")
 def index():
     return "Works :)", 200
+
+#DB for the people opinions/posts
+@app.route("/opinion", methods=["POST", "GET"])
+def opinions_list():
+    if request.method == "POST":
+        body = request.json
+        product_name = body["productName"]
+        product_opinion = body["productOpinion"]
+        product_link = body["poductLink"]
+        _user = body["user"]
+        rating = body["rating"]
+
+        db["opinions"].insert_one(
+            {
+                "productName": product_name,
+                "productOpinion": product_opinion,
+                "productLink": product_link,
+                "_user": user,
+                "rating": rating,
+            }
+        )
+        return jsonify(
+            {
+                "status": "Data is posted to MongoDB!",
+                "productName": product_name,
+                "productOpinion": product_opinion,
+                "productLink": product_link,
+                "_user": user,
+                "rating": rating,
+            }
+        )
+
+    if request.method == "GET":
+        all_data = db["opinions"].find()
+        data_json = []
+        for data in all_data:
+            product_id = data["_id"]
+            product_name = data["productName"]
+            product_opinion = data["productOpinion"]
+            product_link = data["productLink"]
+            user = data["_user"]
+            rating = data["rating"]
+            data_dict = {
+                "id": str(product_id),
+                "productName": product_name,
+                "productOpinion": product_opinion,
+                "productLink": product_link,
+                "_user": user,
+                "rating": rating,
+            }
+            data_json.append(data_dict)
+        print(data_json)
+        return jsonify(data_json)
+
+
+@app.route("/post/<item_id>", methods=["GET"])
```

Comment:
I think a better way to approach the problem is to make something similar to /listings, in where you can get all the reviews for a given product id

Reply:
In the listing it only gets one product, that is why I got the for to get all the opinions of a specific product

Reply:
Oh, right! Sorry haha